astropy/cosmology/parameters.py

```diff
@@ -24,10 +24,10 @@
 The list of cosmologies available are given by the tuple
 `available`. Current cosmologies available:
 
-Planck 15 parameters from Planck Collaboration 2015, arXiv: 1502.01589
+Planck 15 parameters from Planck Collaboration 2016, A&A, 594, A13
```

Comment:
Is it supposed to be ``Planck15`` at the start of the line? (to match the cosmology name)